interpreter/cling/include/cling/Utils/ASTImportSource.h

```diff
@@ -0,0 +1,88 @@
+#ifndef CLING_ASTIMPORTSOURCE_H
```

Comment:
Can you add the usual file header? Copy and adjust it from any other cling header.